ansible/group_vars/all/defaults.yaml

```diff
@@ -45,3 +45,6 @@ kubernetes_bins:
   - kubelet
 
 sysprep: true
+offline_local_bundle_path: ""
+offline_remote_bundle_dir: "/root/.konvoy/offline"
```

Comment:
From standup: Let's think a little bit more about the naming scheme.

There might be conflict between bundles...

(Offline introduces yet another term, can we reuse an existing one?)

Reply:
@dkoshkin @joejulian @faiq @supershal and I talked about this, decided to use `/opt/dkp/packages` for packages (and `/opt/dkp/images` for images. Not yet sure how to change this to fit that model, though...